core/src/mouse/click.rs

```diff
@@ -72,8 +72,11 @@ impl Click {
         } else {
             None
         };
-
-        self.position == new_position
+        let x_consecutive_distance = f32::abs(self.position.x - new_position.x);
+        let y_consecutive_distance = f32::abs(self.position.y - new_position.y);
+        eprintln!("x_distance: {:?}". x_consecutive_distance); 
+        eprintln!("y_distance: {:?}". y_consecutive_distance); 
+        x_consecutive_distance <= 5.0 && y_consecutive_distance <= 5.0
             && duration
                 .map(|duration| duration.as_millis() <= 300)
```

Comment:
Both the timeout and this new distance are hardcoded, would be created if those where env vars/settings.

Reply:
Better to use Point::distance() <= 6.0 like imgui does as it is "cleaner" and a bit less permissive: https://github.com/ocornut/imgui/blob/master/imgui.cpp#L9793